test-suite/non-functional-tests/test-scenario-6/README.md

```diff
@@ -25,6 +25,42 @@ To start the tests, run the following:
 ./start-docker.test.sh
 ```
 
+
+### Load Testing
```

Comment:
So this folder has 6 different "test-scenario" folders. What were your thoughts on that layout @ole4ryb / @azam145 ?

Reply:
So the layout changes are a bit different to the main purpose of this PR which was about testing of PS adapter performance. We will do in a separate PR.